### PR TITLE
Fix typo in BaseHead default loss_factor.

### PR DESCRIPTION
**Pull request by SuX97**

### CODE REVIEW

**Comment by innerlee** [comment]
The problem is deeper. There must be some `**kwargs` not being used and also not being checked

**Comment by SuX97** [comment]
> The problem is deeper. There must be some `**kwargs` not being used and also not being checked

There is no **kwargs used in building heads and losses. The reason why we haven't tested it and not error being revealed is that, in BaseWeightedLoss, loss_weight is set to be 1.0 by default. And in all our usages(e.g. all configs), we override the `loss_cls` by `dict(type='XXXLoss') ` without specifying loss_weight.

**Comment by innerlee** [comment]
Is `loss_factor` a valid argument?

**Comment by SuX97** [comment]
> Is `loss_factor` a valid argument?

Yes, but it will always be overridden together with the total dictionary of `loss_cls`. So we never met this error before.

File: tests/test_models/test_head.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 
 class ExampleHead(BaseHead):
-    # use a ExampleHead to success BaseHead
+    # use a ExampleHead to succeed BaseHead
```
Comment (innerlee) [inline tests/test_models/test_head.py:9]: What does `succeed BaseHead` mean?

Comment (SuX97) [reply tests/test_models/test_head.py:9]: I am not sure, I believe the former author means something like 'inherit'. So I change it to the verb form.

Comment (innerlee) [reply tests/test_models/test_head.py:9]: succeed -> test

**Review by innerlee — APPROVED** [review]

**Pull request merged**